File: 프로그래머스/level2/12949.py
def solution(arr1, arr2):
    a, b, c = len(arr1), len(arr1[0]), len(arr2[0])
    result = [[0] * c for _ in range(a)]

    for i in range(a):
        for j in range(c):
            for k in range(b):
                result[i][j] += arr1[i][k] * arr2[k][j]

    return result

File: 프로그래머스/level2/12949.js
function solution(arr1, arr2) {
    let answer = arr1.map(v => {
        let tmpArr = []
        for (let j = 0; j < arr2[0].length; j++) {
            let tmpNum = 0
            for (let i = 0; i < v.length; i++) {
                tmpNum += arr2[i][j] * v[i]
            }
            tmpArr.push(tmpNum)
        }
        return tmpArr
    })
    return answer
}
console.log(solution([[2, 3, 2], [4, 2, 4], [3, 1, 4]], [[5, 4, 3], [2, 4, 1], [3, 1, 1]]))